sycl/unittests/handler/SetArgForLocalAccessor.cpp

```diff
@@ -1,53 +0,0 @@
-//==------- SetArgForLocalAccessor.cpp --- Handler unit tests --------------==//
-//
-// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
-// See https://llvm.org/LICENSE.txt for license information.
-// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-//
-//===----------------------------------------------------------------------===//
-
-#include <gtest/gtest.h>
-#include <helpers/KernelInteropCommon.hpp>
-#include <helpers/PiMock.hpp>
-
-#include <sycl/sycl.hpp>
-
-// This test checks that we pass the correct buffer size value when setting
-// local_accessor as an argument through handler::set_arg to a kernel created
-// using OpenCL interoperability methods.
-
-namespace {
-
-size_t LocalBufferArgSize = 0;
-
-pi_result redefined_piKernelSetArg(pi_kernel kernel, pi_uint32 arg_index,
-                                   size_t arg_size, const void *arg_value) {
-  LocalBufferArgSize = arg_size;
-
-  return PI_SUCCESS;
-}
-
-TEST(HandlerSetArg, LocalAccessor) {
```

Comment:
This [comment](https://github.com/intel/llvm/pull/12172/files#r1481365200) gives more context on what happened to this kind of tests.

This test is covered in this e2e [test](https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/DeprecatedFeatures/set_arg_interop.cpp).